### ISSUE TITLE
bug with next link and a11y anchor

### ISSUE THREAD

**Issue by ShooTeX** [21stdigital/eslint-config-21st#3]
```
            <Link as={backPath} href="/themen/[topic]/[...slug]">
              <a>
                <BackButton>Zurück</BackButton>
              </a>
            </Link>
```

> The href attribute is required for an anchor to be keyboard accessible. Provide a valid, navigable address as the href value. If you cannot provide an href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.mdeslintjsx-a11y/anchor-is-valid
